media/inf/syssetup.inf

```diff
@@ -144,3 +144,22 @@ Control="System\CurrentControlSet\Control"
 
 [Classes]
 HKLM,%CurrentVersion%\Installer,"InstallerLocation",,"%11%"
+
+[DestinationDirs]
+IECopy=16422,Internet Explorer
+
+[Finalize] ; Executed by RunOnceEx on first normal boot
+; FIXME: This should be moved to ie.inf,DefaultInstall and executed by [Infs.Always]
+AddReg=FinalizeAddReg
+CopyFiles=IECopy
+RegisterDlls=IEReg
+
+[FinalizeAddReg]
+HKLM,SOFTWARE\Microsoft\.NETFramework,InstallRoot,,"%10%\Microsoft.NET\Framework\"
+
+[IECopy]
+iexplore.exe,..\iexplore.exe,,0x00002000
+
+[IEReg]
+16422,Internet Explorer,iexplore.exe
+11,,rundll32.exe,,99,"advpack.dll,DelNodeRunDLL32 ""%10%\iexplore.exe"""
```

Comment:
```suggestion
11,,rundll32.exe,,60,"advpack.dll,DelNodeRunDLL32 ""%10%\iexplore.exe"""
```

Reply:
I fixed the default timeout handling instead